code/script.js

```diff
@@ -34,18 +43,282 @@ const showMessage = (message, sender) => {
+const handleNameInput = (event) => { 
+  event.preventDefault(); 
+  //name saved for later
+  name = nameInput.value
+  console.log(name)
+  //the users name pops up as a message
+  showMessage(name, "user")
+  //name input is reset
+  nameInput.value = ""; 
+  //after 1s bot asks for gender
+  setTimeout(() => askForGender(name), 1000);
+}
+
+const askForGender = (name) => {
+  //Bot's welcome message with name pops up. Asks for gender.
+  showMessage(`Nice to meet you ${name}! <br><br> Are you looking for a gift for a gentleman or a lady?`, "bot")
+  //Buttons with gender choice appears
+  inputWrapper.innerHTML =`
+  <button class="gender-btn" id="lady-btn">👩‍🦰</button>
+  <button class="gender-btn" id="gentleman-btn">🧔</button>
+  `; 
+  //Adding eventlisteners to gender buttons
+  document.getElementById('lady-btn')
+    .addEventListener('click', () => {
+      //redeclaring global variable "gender"
+      gender = "lady"
+      //User reply comes up
+      showMessage("It's for a lady.", "user")
+      //Invokes bot's next message
+      setTimeout(() => confirmGender(gender), 1000)
+    })
+  document.getElementById('gentleman-btn')
+    .addEventListener('click', () => {
+      //redeclaring global variable "gender"
+      gender = "gentleman"
+      //User reply comes up
+      showMessage("It's for a gentleman.", "user")
+      //Invokes bot's next message
+      setTimeout(() => confirmGender (gender), 1000)
+    })
+}
+
+const confirmGender = (gender) => {
+  //if it's a lady the bot will confirm this
+  if (gender === "lady") {
+    showMessage (`${name}, I'm sure we'll find her a lovely gift!`, 'bot')
+    //if it's a gentleman the bot will confirm this
+  } else if (gender === "gentleman") {
+    showMessage (`${name}, I'm sure we'll find him a lovely gift!`, 'bot')
+  }
+  //invoking bot's next message about price
+  setTimeout(() => askPriceRange(), 1000)
+}
+
+const askPriceRange = () => {
+  showMessage (`So, how much is this special gift supposed to cost?`, 'bot')
+  //Drop down window with three options will show up
+  inputWrapper.innerHTML =`
+    <select id="price-select">
+    <option value="">--Please choose a price range--</option>
+    <option id="range-1" value="range-1">10-50€</option>
+    <option id="range-2" value="range-2">50-100€</option>
+    </select>
+    `; 
+    select = document.getElementById("price-select")
+  const priceRangeBtns = inputWrapper.innerHTML
+  //Adding eventlisteners to option range 1
+  document.getElementById('price-select')
+    .addEventListener('change', (event) => {
+      //Storing the value of the event target by redeclaring global variabel
+      selectedPriceRange = event.target.value
+      //User reply comes up
+      showMessage("This is what I can afford.", "user")
+      select.remove() //Removing the buttons to make the chat look closed.
+      //Invokes bot's next message
+      setTimeout(() => confirmPriceRange(selectedPriceRange), 1000)
+    })
+}
```

Comment:
I'm not entirely sure, but I think you could write this eventListener using the declared variable `select` instead of `document.getElementById('price-select')`, since you've already "fetched" it once - to allow for a cleaner code, so like this:
```
select.addEventListener('change', (event) => {
// your code here
}
```

As I said, I haven't tried this out myself, there might be issues due to the select being removed later in the function, but you could try it?